sympy/printing/tests/test_llvmjit.py

```diff
@@ -0,0 +1,45 @@
+
+from sympy.external import import_module
+
+
+if import_module('llvmlite'):
+    import sympy.printing.llvmjitcode as g
+else:
+    disabled = True
```

Comment:
I just didn't see the use of disabled below, so I don't think it actually disables the tests.